guide/like.md

```diff
@@ -0,0 +1,10 @@
+## User Handbook
+### Liking a post
```

Comment:
Might want to note somewhere here that it requires the Likes extension to be enabled.